### DISCUSSION TITLE
using github as a source of truth for government data

### DISCUSSION THREAD

**Discussion by christopherreay** [community/community#122624] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

HI :)

so... we have a decently innovative blockchain solution for government transparency. The question I wish to discuss (is in the title), however the specific point is this:

we will publish a bunch of government interactions onto some public blockchain. however, there are many more events/log moments that will exist between the items that are published onto the blockchain. I think it is a good  idea to use github as a source of truth for the intermediary events, *and* as a proof of hash embedded in the blockchain items (NFTs).... 

so my question is, is this an "abuse" of github (the actual amount of data is relatively small)... or is it a good and reasonable use of the platform?

**Comment by Pumpiron4u** [comment]
Your approach to utilizing GitHub as a source of truth for intermediary events in your blockchain solution for government transparency raises an interesting question. While the actual amount of data may be relatively small, it's essential to consider whether this usage aligns with GitHub's intended purpose and terms of service. On one hand, GitHub provides version control and collaboration tools ideally suited for managing and tracking code changes. However, leveraging it for non-code-related data storage, even if the volume is minimal, could potentially be seen as straying from its intended use. It's worth exploring alternative platforms or considering how this approach could be optimized within GitHub's framework to ensure compliance and efficiency. What are your thoughts on striking the right balance between leveraging GitHub's capabilities and respecting its guidelines?

**Reply by christopherreay** [reply]
my thoughts are "I thought I would ask".
